### DISCUSSION TITLE
Replicaset history in the cluster is being tracked by argocd

### DISCUSSION THREAD

**Discussion by steffenneyt** [argoproj/argo-cd#14928] [General]
Our cluster keeps track of 10 replicasets in total. 1 active and 9 history replicasets, but the problem that we have right now is that argocd is also tracking the history replicasets. Is there any way that we can configure argocd to not track those? Because its causing some clutter in the argocd UI.

**Comment by crenshaw-dev** [comment]
There's an open issue tracking this. No complete solution yet: https://github.com/argoproj/argo-cd/issues/12196
